modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util/store/ioRequest.es.js

```diff
@@ -0,0 +1,14 @@
+export default function ioRequest(config) {
```

Comment:
We don't need this file, just use `fetch`. Import fetch with relative path from this module.